src/lwc/rd2EntryForm/rd2EntryForm.js

```diff
@@ -218,6 +221,21 @@ export default class rd2EntryForm extends LightningElement {
         this.isElevateWidgetDisplayed = this.isElevateCustomer === true
             && !this.isEdit // The Elevate widget is applicable to new RDs only
             && paymentMethod === 'Credit Card';// Verify the payment method value
+
+        if (this.isElevateWidgetDisplayed === true) {
+            // TO-DO: Prepopulate the Cardholder Name with the currently selected Contact or Organization Name
```

Comment:
I am debating if we should keep this commented out code, but I guess it is fine since the next WI for the cardholder name should be worked on relatively soon.

Reply:
I left it in only because I had started working on it while it was still part of this WI. Now that it's moved into the other WI, I figured it might still be useful here.